plugins/utils_operators/file_operator.py

```diff
@@ -23,48 +23,91 @@ class DownloadFileOperator(BaseOperator):
     def __init__(
         self,
         file_url: str,
-        dir_task_id: str,
+        dir: str,        
```

Comment:
Minor, but what's with all the whitespaces after this line? 😆 

Have you enabled auto-formatting for Python in VS Code yet? I use `black`. This is also why I've found `pre-commit` helpful: catch these issues before they ever make it to the repo. Consider it!

Reply:
lmfao not a clue - I'll fix that and then look into black or pre-commit (which, as you can see, I still haven't done yet)